main.py

```diff
@@ -0,0 +1,46 @@
+import SLAM.core
+from SLAM.landmark_extractor import extract_spike_landmarks
+from SLAM.landmark_matcher import match_landmarks
+from SLAM.position_evaluator import evaluate_position
+from SLAM.receiver import Receiver
+from multiprocessing import Queue
+from SLAM.globalizer import *
+from SLAM.sender import *
```

Comment:
Can you import the functions you want to use directly, instead of importing everything.